FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -654,7 +624,7 @@ extension ComposeViewController {
 // MARK: - Action Handling
 extension ComposeViewController {
     private func searchEmail(with query: String) {
-        googleService.searchContacts(query: query)
+        contactsProvider.searchContacts(query: query)
```

Comment:
Is this searching local contacts or external / google contacts? We may want to distinguish for clarity, something like:

`LocalContactsProvider` vs `CloudContactsProvider` ?